s4e-web/src/app/app.configuration.ts

```diff
@@ -18,6 +18,14 @@
 export interface IRemoteConfiguration {
   osmUrl: string;
   geoserverUrl: string;
+  helpdesk?: {
+    type: string;
+    href: string;
+  };
+  analytics?: {
+    type: string;
+    href: string;
+  };
```

Comment:
It may be useful to document the configuration in the README.md, as I only put a generic description there.